cmd/src/cli/js-tests-scaffold/README.md

```diff
@@ -0,0 +1,31 @@
+# js-tests-scaffold
```

Comment:
These files aren't from what's currently in `master` branch of `js-tests-scaffold`... it'd be better if you used those

Reply:
I tried and I can't yet because master is what works for holochain-nodejs.  For hcshell it needs to be this version.  I'll update as soon as we merge nodejs stuff in.

Reply:
Ok fair, I see